runtime/integration-tests/src/generic/cases/loans.rs

```diff
@@ -42,112 +47,323 @@ use crate::{
 const POOL_ADMIN: Keyring = Keyring::Admin;
 const INVESTOR: Keyring = Keyring::Alice;
 const BORROWER: Keyring = Keyring::Bob;
+const LOAN_ADMIN: Keyring = Keyring::Charlie;
+const ANY: Keyring = Keyring::Dave;
```

Comment:
Nit: We should make it clear this refers to a substrate address
```suggestion
const ANYONE: Keyring = Keyring::Dave;
```